server/controllers/finance/reports/incomeExpense/report.handlebars

```diff
@@ -38,16 +38,16 @@
                 <tr>
                   <td>{{number}}</td>
                   <td>{{label}}</td>
-                  <td class="text-right">{{currency debit}}</td>
-                  <td class="text-right">{{currency credit}}</td>
+                  <td class="text-right">{{currency debit ../enterprise.currency_id}}</td>
```

Comment:
Just FYI, I believe the enterprise is always available on the `metadata` object.  For example, 
```html
<td class="text-right">{{currency debit metadata.enterprise.currency_id}}</td>
```

You can see this usage in [the invoices report](https://github.com/IMA-WorldHealth/bhima-2.X/blob/master/server/controllers/finance/reports/invoices/report.handlebars#L45).